lib/seek/samples/attribute_type_handlers/linked_custom_metadata_attribute_type_handler.rb

```diff
@@ -0,0 +1,17 @@
+module Seek
+  module Samples
+    module AttributeTypeHandlers
+      class LinkedCustomMetadataAttributeTypeHandler < BaseAttributeHandler
+
+        def test_value(array_value)
+
+        end
+
+        def convert(value)
+
+        end
```

Comment:
this looks wrong, they override the parent class methods, but will just return nil